src/plugins/intel_cpu/src/nodes/executors/executor_factory.hpp

```diff
@@ -95,104 +67,41 @@ class ExecutorFactory {
     }
 
     /**
-     * @brief Preconfigures an executor based on the provided memory arguments.
-     *
-     * Preconfigures an executor by selecting an appropriate implementation based on the provided
-     * memory arguments and by creating an executor using the implementation.
-     *
-     * @param memory The memory parameters used for selecting the appropriate executor implementation.
-     *
-     * @note The main use case is to offload executor data preparation (i.e. weights packing)
-     *       From the make() call
-     * @todo Currently supports creating a single executor.
-     *       For some nodes it can be worth to preconfigure all the executors.
-     */
-    void preconfigure(const MemoryArgs& memory) {
-        executor::Config<Attrs> config{memoryDescsFromMemory(memory), m_attrs, m_postOps};
-
-        cacheFallbackStatus(config);
-
-        const size_t implId = select(memory, 0);
-        const auto& impl = m_suitableImplementations[implId].get();
-        DEBUG_LOG("Preconfiguring executor: ", impl.name());
-
-        if (m_implementationRequiresFallback[implId]) {
-            if (auto fallbackConfig = impl.requiresFallback(config)) {
-                fallback<Attrs, NodeT>(config, *fallbackConfig, memory, m_context, impl.name());
-            }
-        }
-
-        (void)create(implId, memory, m_context);
-    }
-
-    /**
-     * @brief Creates an Executor instance based on provided memory arguments.
-     *
-     * Creates an Executor instance using the provided MemoryArgs, selecting an appropriate implementation
-     * based on the characteristics of the memory. It handles fallback scenarios if necessary and updates the executor
-     * with the given memory information.
+     * @brief Creates an Executor instance based on the provided \memory arguments.
```

Comment:
`\memory` -> `memory`

Reply:
I was trying to follow a doxygen syntax for referencing function arguments.
But it does not really matter, probably.
Removed